app/views/dashboard_v2/_alerts.html.erb

```diff
@@ -0,0 +1,14 @@
+<%# Welcome a new user %>
+<% if current_user && Time.now - current_user.created_at < 1.week %>
```

Comment:
Should we only show this alert if the person has joined in last 1 week and has not posted any node? What do you think?

Reply:
I think we can... what does it imply when a new user has posted a node in relation to the Welcome message?

Reply:
i think we used to check if someone posted, but i think even if they have posted we ought to show the welcome message. It'll only bother them for a few days :-)

Reply:
@jywarren tagging you in this discussion with @Sagarpreet about adding an extra condition